### DISCUSSION TITLE
find_eqpt for discrete-time system

### DISCUSSION THREAD

**Discussion by Maksymilian-Plywaczyk** [python-control/python-control#819] [Q&A] [answered]
Hi,

I found that find_eqpt function is not supported with discrete-time systems. It is any possible way to solve the equilibrium points for an input/output discrete system in Control library? Thanks a lot for your attention and help :)

**Comment by murrayrm** [comment, marked as answer]
This is now implemented (PR #824) and is part of control-0.9.3.
